src/components/Fund/DropDownMenu.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Fragment } from 'react'
+import { Menu, Transition } from '@headlessui/react'
+import dropDownArrow from "../../img/icons/dropDownArrowGray.svg";
+
+function classNames(...classes: string[]) {
+    return classes.filter(Boolean).join(' ')
+}
+
+type DropDownMenuProps = {
+    handleFundSwitch: any;
+    presentFund: any,
+    funds: any,
+    presentFundId: number,
+}
+
+const DropDownMenu: React.FC<DropDownMenuProps> = ({ handleFundSwitch, presentFund, funds, presentFundId }) => {
```

Comment:
We should name the components better. This is a dropdown that shows the list of funds right? Then let's call it FundDropdown. DropdownMenu makes it seem like it's a generic dropdown, which it is not.

Why is Menu a better option here instead of ListBox? 

we should also consider wrapping the Menu / ListBox in our own components and style everything. this would be a generic Dropdown Component. Then when we create a FundDropdown, we can just pass in the data.

Reply:
Yeah, I have already used ListBox, just didn't delete the DropdownMeny yet. Will clean all this up in the next push. And also making LisBox as generic as possible so that we can reuse it whenever needed.